### PR TITLE
Add compat data for NavigatorPlugins

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/NavigatorPlugins

### CODE REVIEW

File: api/NavigatorPlugins.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": null,
+            "standard_track": null,
+            "deprecated": null
```
Comment (ExE-Boss) [inline api/NavigatorPlugins.json]: None of these can be `null`, they must be either `true` or `false` (I recommend copying the data from the Basic support entry)

File: api/NavigatorPlugins.json [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": null,
+              "standard_track": null,
+              "deprecated": null
```
Comment (ExE-Boss) [inline api/NavigatorPlugins.json]: ditto

File: api/NavigatorPlugins.json [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/NavigatorPlugins/mimeTypes",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/NavigatorPlugins.json]: All three versions of Chrome should be `true`.

File: api/NavigatorPlugins.json [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/NavigatorPlugins/plugins",
+            "support": {
+              "webview_android": {
+                "version_added": null
+              },
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": null
```
Comment (jpmedley) [inline api/NavigatorPlugins.json]: All three versions of Chrome should be `true`.

File: api/NavigatorPlugins.json [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/NavigatorPlugins/javaEnabled",
+            "support": {
+              "webview_android": {
+                "version_added": null
+              },
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": null
```
Comment (jpmedley) [inline api/NavigatorPlugins.json]: All versions of three should be `true`.

File: api/NavigatorPlugins.json (hunk 1 above)
Comment (maboa) [reply api/NavigatorPlugins.json]: 👍

File: api/NavigatorPlugins.json (hunk 2 above)
Comment (maboa) [reply api/NavigatorPlugins.json]: 👍

File: api/NavigatorPlugins.json (hunk 3 above)
Comment (maboa) [reply api/NavigatorPlugins.json]: 👍

File: api/NavigatorPlugins.json (hunk 4 above)
Comment (maboa) [reply api/NavigatorPlugins.json]: 👍

File: api/NavigatorPlugins.json (hunk 5 above)
Comment (maboa) [reply api/NavigatorPlugins.json]: 👍

**Comment by maboa** [comment]
Updated :)

**Pull request merged**